doc/classes/GDExtensionManager.xml

```diff
@@ -56,6 +56,18 @@
 		</method>
 	</methods>
 	<signals>
+		<signal name="extension_loaded">
+			<param index="0" name="extension" type="GDExtension" />
+			<description>
+				Emitted after the editor has finished loading a new extension.
```

Comment:
This appears to only be emitted in editor builds - this should probably be noted in the docs.

Reply:
Done!